docs/pages/versions/unversioned/sdk/localization.md

```diff
@@ -57,24 +57,27 @@ Let's make our app support English and Japanese.
 import * as React from 'react';
 import { View, StyleSheet, Text } from 'react-native';
 import * as Localization from 'expo-localization';
-import i18n from 'i18n-js';
+import { I18n } from 'i18n-js';
 
 // Set the key-value pairs for the different languages you want to support.
-i18n.translations = {
+const translations = {
   en: { welcome: 'Hello', name: 'Charlie' },
   ja: { welcome: 'こんにちは' },
 };
+const i18n = new I18n(translations);
+
 // Set the locale once at the beginning of your app.
-i18n.locale = Localization.locale;
+i18n.locale = 'ja';
```

Comment:
I am not sure I get this change. Shouldn't this be `i18n.locale = Localization.locale;`? Otherwise, why would one use `expo-localization` library in this example?

Reply:
This example demonstrate the language fallback feature. I will add another label with device language in View  and comment for the clarity.